### PR TITLE
Add Exact Online

### PR DESCRIPTION
**Pull request by jeohist**
I know the CONTRIBUTING.MD says not to add multiple URLs, but since this is one website hosted on multiple URLs for different countries and we'd really like them to be shown in 1Password's Watchtower functionality... 

I don't think there's really a way for this to be more elegant, other than implementing something like the YAML files in #1025 or straight-up allowing multiple URLs for one website (or Exact Online hosting everything in one domain).

### CODE REVIEW

**Comment by mxxcon** [comment]
Are all those sites use the same login system or are they all separate?
If it's all the same then we should not have multiple entries, just a single main one is sufficient.

If you want 1pass' watchtower to display green checkmark or whatever, then you should approach them to implement the concept of related domains. Other password managers have it and it works great. I would not want us to mangle our data to compensate for deficiencies of 1pass.

**Comment by jeohist** [comment]
They are all separate login systems (i.e. they don't share usernames or passwords and the TOTP token is unique for each site).


________________________________

**Comment by jeohist** [comment]
Is anything required from my side at this point in order to get this merged?

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
For the `url` tag we list the homepage of the site and not the login page..
For star.exactonline.* the main page seems to be exact.com so please change the urls to that. Furthermore since they all would have the same url after the edit mentioned above I suggest you only use one entry instead of the 7 you added.

For the `doc` tags we require a url that isn't behind a login-page so if there's no other link that's open to the public then remove the tag.

We'll need some sort of confirmation that they indeed support 2FA so please send a screenshot of the login process or documentation page (with personal information blurred out or painted over)

**Comment by jeohist** [comment]
Thanks for the response @Carlgo11.

start.exactonline.* is the website of the Exact Online application. The only reason exactonline.* redirects to exact.com is a marketing thing. Here's how it looks on the inside: https://i.imgur.com/GKp9MjB.png
Consider it the same situation as GMail :-)

With regards to the documentation pages: Those work fine here without logging on, they are publicly available. I do notice a few of the applications have a broken URL, I'll fix that. 

Login process:
Login page: https://i.imgur.com/fZyaC3k.png
2FA: https://i.imgur.com/DObnyVH.png

**Comment by Carlgo11** [comment]
@jeohist what would you say the main page of the company/services are?

**Comment by jeohist** [comment]
Since #3414 was implemented, I will simplify this PR by only including start.exactonline.nl. The others are not (yet) in the Alexa Top 200k. 

And in the meantime, it'll be a fun pet project to think about #1025 :-)

**Review by mxxcon — DISMISSED** [review]

File: _data/finance.yml [hunk 1]
```diff
@@ -53,6 +53,13 @@ websites:
       img: ditech.png
       tfa: No
 
+    - name: Exact Online
+      url: https://start.exactonline.nl
```
Comment (Carlgo11) [inline _data/finance.yml]: One last thing, for the url we list the main page of a site. That page can't be a login page and so I suppose the url value would be https://www.exact.com/

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: _data/finance.yml (hunk 1 above)
Comment (jeohist) [reply _data/finance.yml]: start.exactonline.* is the website of the Exact Online application. The only reason exactonline.* redirects to exact.com is a marketing thing. Here's how it looks on the inside: https://i.imgur.com/GKp9MjB.png
Consider it the same situation as GMail :-)

Comment (Carlgo11) [reply _data/finance.yml]: Still, we don't list login pages in the `url` field so either make start.exactonline.nl redirect to something else than a login page or change the `url` value to https://www.exact.com/.

**Comment by jeohist** [comment]
I'm part of the team maintaining the website. Literally any request you send to the Exact Online application is sent to start.exactonline.nl. I don't see what the problem is here, it's not just a login page, it's the entire bloody application.

**Comment by Carlgo11** [comment]
@jeohist The `url` tag is designed to list a sites main page. If it's a login page then it's not a main page (or at least not 99.99% of the time.)

If a new user wants to see what a service does then the first thing they'll look at is the main page and not the login page.
If you didn't know anything about Exact Online what would this tell you? My guess is not a lot.
screenshot

File: _data/finance.yml [hunk 2]
```diff
@@ -53,6 +53,13 @@ websites:
       img: ditech.png
       tfa: No
 
+    - name: Exact Online
+      url: https://exactonline.nl
```
Comment (Carlgo11) [inline _data/finance.yml]: chrome_2018-10-27_21-25-54
And the site does a 301 to exact.com
```suggestion
      url: https://exact.com/
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — DISMISSED** [review]

**Review by brycx — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**